os/arch/arm/src/artik05x/src/artik05x.h

```diff
@@ -57,6 +57,11 @@
  * Included Files
  ****************************************************************************/
 
+#ifdef CONFIG_SMARTFS_MULTI_ROOT_DIRS
+#undef CONFIG_ARTIK05X_AUTOMOUNT_USERFS_DEVNAME
+#define CONFIG_ARTIK05X_AUTOMOUNT_USERFS_DEVNAME "/dev/smart0p8d1"
```

Comment:
same as previous.
Because we provide this as a config, user can change a name of mount point. But this code does not concern it.
We can not gaurantee user will set it as ```/dev/smart0p8```.
Let's define and use new definition like ARTIK05X_AUTOMOUNT_USERFS_DEVNAME and set it like tc does.